### PR TITLE
release ws-funder-ner@1.0.0

### PR DESCRIPTION
**Pull request by Luc-Ank**
Merge avec la branche principale ws-funder-ner

### CODE REVIEW

File: applications/ws-fund-ner/config.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+		"NODE_OPTIONS": "--max_old_space_size=1024",
+		"NODE_ENV": "production",
+		"PIP_NO_DEPENDENCIES": 0,
+		"PIP_DISABLE_PIP_VERSION_CHECK": 1,
+		"PIP_NO_CACHE_DIR": 0
+	},
+	"packages": [
+		"@ezs/core@2.4.3",
+		"@ezs/spawn@1.2.30",
+		"@ezs/basics@1.23.3"
```
Comment (parmentf) [inline applications/ws-fund-ner/config.json:18]: These are old versions.

```suggestion
		"@ezs/core@3.3.0",
		"@ezs/spawn@1.4.5",
		"@ezs/basics@2.5.0"
```

File: applications/ws-fund-ner/package-app.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+	"private": true,
+	"name": "ws-funder-ner",
+	"dependencies": {
+		"@ezs/core": "2.4.3",
+		"@ezs/basics": "1.23.3",
+		"@ezs/spawn": "^1.2.30",
+		"dotenv-cli": "4.1.1",
+		"npm-watch": "0.11.0"
```
Comment (parmentf) [inline applications/ws-fund-ner/package-app.json:9]:
```suggestion
		"@ezs/core": "3.3.0",
		"@ezs/basics": "2.5.5",
		"@ezs/spawn": "1.4.5",
		"dotenv-cli": "7.3.0",
		"npm-watch": "0.11.0"
```

File: applications/ws-fund-ner/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Before calling `npm run build` or `npm start`.
+
+> 📗 Suggestion: put them in `.env` file and don't forget to run `source .env`
+> before calling `npm run build` or `npm start`.
+
+## Test
+
+```bash
+curl -X POST "http://localhost:31976/v1/funderTagger/funderTagger" \
```
Comment (parmentf) [inline applications/ws-fund-ner/README.md:22]: This has more to do with <https://gitbucket.inist.fr/tdm/web-services/tree/master/fund-ner>, but this route is a bit complicated.

I'd prefer `v1/funderTagger`, or even `v1/tagger`, as the domain name will include `funder-ner`.
<https://funder-ner.services.istex.fr/v1/tagger> should be enough, shouldn't it?.

**Review by parmentf — CHANGES_REQUESTED** [review]

File: applications/ws-fund-ner/config.json [hunk 4]
```diff
@@ -13,8 +13,8 @@
 		"PIP_NO_CACHE_DIR": 0
 	},
 	"packages": [
-		"@ezs/core@2.4.3",
-		"@ezs/spawn@1.2.30",
-		"@ezs/basics@1.23.3"
+		"@ezs/core@3.3.0",
+		"@ezs/spawn@1.4.5",
+		"@ezs/basics@2.5.0"
```
Comment (parmentf) [inline applications/ws-fund-ner/config.json:18]: Oops, `@ezs/basics` last version is 2.5.5.

File: applications/ws-fund-ner/README.md (hunk 3 above)
Comment (Luc-Ank) [reply applications/ws-fund-ner/README.md]: I changed path in both gitbucket and github repository to v1/tagger .

**Review by parmentf — APPROVED** [review]

**Pull request merged**